docs/webaverse/parcels.md

```diff
@@ -3,6 +3,8 @@ id: parcels
 title: Parcels 
 ---
 
-Parcels represent an area of the Webaverse world that you can own and build whatever you wish on.
+Parcels represent an area of the Webaverse world that you can own and build whatever you wish on. Size is not a limitation, you can build full virtual experiences that can be hyperlinked from The Street.
```

Comment:
The Street needs to be a `.md`. Some details are in `seasons.md` but we need a much higher level and image-filled explainer of the street concept.